Lib/dataclasses.py

```diff
@@ -749,12 +749,19 @@ def _get_field(cls, a_name, a_type):
 
     return f
 
+def _set_qualname(cls, value):
+    # Ensure that the functions returned from _create_fn uses the proper
+    # __qualname__ (the class they belong)
```

Comment:
This would read better as `(the class they belong to).` Add "to" and the trailing dot. I realize this is a nit. I also realize some people will object to ending a phrase with a preposition. I'm okay with it.

Reply:
Thank you for the suggestions, addressed.